### PR TITLE
Hide the Era Points from the validator page

### PR DESCRIPTION
**Pull request by piotrMocz**
They were confusing and seldom understood.
Screenshot from 2022-12-22 16-58-15

### CODE REVIEW

File: packages/page-staking/src/Query/Validator.tsx [hunk 1]
```diff
@@ -12,18 +12,25 @@ import ChartPoints from './ChartPoints';
...
     <Columar className={className}>
-      <Columar.Column>
-        <ChartPoints validatorId={validatorId} />
-        <ChartRewards validatorId={validatorId} />
-      </Columar.Column>
       <Columar.Column>
         <ChartStake validatorId={validatorId} />
         <ChartPrefs validatorId={validatorId} />
       </Columar.Column>
+      <Columar.Column>
```
Comment (kostekIV) [inline packages/page-staking/src/Query/Validator.tsx:32]: whye the change in the order of Columar.Column?

**Review by kostekIV — APPROVED** [review]
Other than lints from check lgtm :)

**Review by Marcin-Radecki — APPROVED** [review]

**Review by DamianStraszak — APPROVED** [review]

**Comment by youPickItUp** [comment]
I've merged newest changes and simplified the logic slightly

File: packages/page-staking/src/Query/Validator.tsx (hunk 1 above)
Comment (youPickItUp) [reply packages/page-staking/src/Query/Validator.tsx:32]: I guess it's to avoid column switching place depending on selected chain

**Pull request merged**